### ISSUE TITLE
installer: Don't install yara manually

### ISSUE THREAD

**Issue by maringuu** [fkie-cad/FACT_core#864]
Ideally we should install yara from the repos of the respective distro.

**Comment by jstucke** [comment]
I disagree: The different distros have vastly differing versions. E.g. Ubuntu 18.04 (which we still support) has YARA 3.7.1 which is not only pretty old, but also has a slightly different API which causes unexpected problems (which we probably don't want to handle). What we probably should do, though, is pinning the version so the latest stable one so that we are not surprised by additional API changes.
